tests/searchcommands/test_search_command.py

```diff
@@ -33,8 +33,21 @@
 import os
 import re
 
+from io import TextIOWrapper
+
 import pytest
 
+def build_command_input(getinfo_metadata, execute_metadata, execute_body):
+    header = ('chunked 1.0,{},0\n{}'.format(len(six.ensure_binary(getinfo_metadata)), getinfo_metadata) +
```

Comment:
`header` seems like an odd name for the string that contains the entire command input. 

I really like seeing this split out into a function though. :+1:

Reply:
Renamed.